worksheet_B/Mandelbrot.cpp

```diff
@@ -1,8 +1,8 @@
-// Mandelbrot.cpp : Defines the entry point for the console application.
-//
-
 #include "stdafx.h"
 
+float ImageWidth = 800.0;
+float ImageHeight = 800.0;
```

Comment:
Good idea to pull these out as constants. However you should ideally also change the image constructor to use these values, i.e. this line:
```
CImg<unsigned char> image(800, 800, 1, 3, 0);
```